### DISCUSSION TITLE
ZTM Access

### DISCUSSION THREAD

**Discussion by YemiYusuf** [mrdbourke/zero-to-mastery-ml#112] [Q&A]
I was wondering, should the Udemy course automatically guarantee or give Students access to ZTM? Or do I still need to register and pay separately for the ZTM account despite having registered for your Udemy course?

**Comment by mrdbourke** [comment]
Hey Yusuf, Welcome to the course!In short: Udemy + ZTM are separate.Udemy hosts ZTM courses available for single purchase.Where as ZTM hosts *all* ZTM courses available for single purchase/via membership (membership gives access to all courses). Best,Daniel BourkeOn 6 Dec 2024, at 9:18 am, Yusuf ***@***.***> wrote:﻿
I was wondering, should the Udemy course automatically guarantee or give Students access to ZTM? Or do I still need to register and pay separately for the ZTM account despite having registered for your Udemy course?

**Reply by YemiYusuf** [reply]
Okay Daniel. Thanks for the prompt response.
